File: SofiForce.Models/Models/EntityModels/SalesErrorModel.cs
using System;

namespace Models
{
    public class SalesErrorModel
    {
        public int SalesErrorId { get; set; }
        public string SalesErrorCode { get; set; }
        public string SalesErrorNameEn { get; set; }
        public string SalesErrorNameAr { get; set; }
        public string SalesErrorTemplateAr { get; set; }
        public string SalesErrorTemplateEn { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/SalesErrorModel.ts
import { SalesOrderErrorModel } from "./salesOrderErrorModel";

export interface SalesErrorModel {
    salesErrorId: number;
    salesErrorCode: string;
    salesErrorNameEn: string;
    salesErrorNameAr: string;
    icon: string;
    color: string;
    displayOrder: number | null;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}